common/packages/tinyscheme/scheme.c

```diff
@@ -4095,24 +4095,25 @@ static pointer opexe_4(scheme * sc, enum scheme_opcodes op)
     case OP_ERR0:		/* error */
 	sc->retcode = -1;
 	if (!is_string(car(sc->args))) {
-	    sc->args = cons(sc, mk_string(sc, " -- "), sc->args);
+            sc->args = cons(sc, mk_string(sc, " -- "), sc->args);
 	    setimmutable(car(sc->args));
 	}
-	putstr(sc, "Error: ");
+        if (sc->interactive_repl)
+            putstr(sc, "Error: ");
```

Comment:
Looks like this entire file needs to be re-tabbed/spaced.

Reply:
It does. I'm not fixing tinyscheme in this commit, however.